lua/nvim_lsp/julials.lua

```diff
@@ -0,0 +1,30 @@
+local skeleton = require 'nvim_lsp/skeleton'
+local util = require 'nvim_lsp/util'
+local lsp = vim.lsp
+
+skeleton.julials = {
+  default_config = {
+    cmd = {
+        "julia", "-e", [[ using LanguageServer; p = joinpath(dirname(dirname(pathof(LanguageServer))), "contrib", "languageserver.sh"); run(`$p`); ]]
```

Comment:
This seems like it assumes linux via the `.sh` file. This was the init sequence I used from many months ago. It may have been updated, but if not then I think it would be better. Also `[[ ]]` blocks can span multiple lines, so you can avoid having to smoosh it on one line. It's more readable that way.

```julia
        using Pkg;
        using LanguageServer;
        import StaticLint;
        import SymbolServer;
        env_path = dirname(Pkg.Types.Context().env.project_file);
        debug = false;

        server = LanguageServer.LanguageServerInstance(stdin, stdout, debug, env_path, "", Dict());
        server.runlinter = true;
        run(server);
```

Reply:
I changed it to using your script and using `[[ ... ]]` multi line strings in lua.